<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFileDialog</class>
 <widget class="QDialog" name="LoadFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkLoadRange">
     <property name="text">
      <string>Load only range of bytes:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>From:</string>
        </property>
        <property name="buddy">
         <cstring>rangeStart</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="IntegerEdit" name="rangeStart" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Length:</string>
        </property>
        <property name="buddy">
         <cstring>rangeLength</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="IntegerEdit" name="rangeLength" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblLoadSize">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkFreezeSize">
     <property name="text">
      <string>Freeze size</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkMemoryLoad">
     <property name="text">
      <string>Completely read into memory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkReadOnly">
     <property name="text">
      <string>Read only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntegerEdit</class>
   <extends>QWidget</extends>
   <header>hex.integeredit</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkLoadRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
